lib/wings/model_transformer.rb

```diff
@@ -54,7 +54,8 @@ def self.relationship_keys_for(reflections:)
                       .keys
                       .reject { |k| k.to_s.include?('id') }
                       .map { |k| k.to_s.singularize + '_ids' }
-      relationships.delete(:member_ids) # Remove here.  Members will be extracted as ordered_members in attributes method.
+      relationships.delete('member_ids') # Remove here.  Members will be extracted as ordered_members in attributes method.
+      relationships.delete('ordered_member_proxy_ids') # This does not have a Valkyrie equivalent.
```

Comment:
'ordered_member_proxy_ids' needs to be deleted too.  It does not follow the same patters as the other '_ids' attributes.  And I don't believe there is a Valkyrie equivalent for this as members are ordered in Valkyrie and don't use a proxy to do it.